### PR TITLE
Add FAQ: lingering agent pods.

### PR DESCRIPTION
**Pull request by t4lz**
Feel free to add commits to this branch if you want to.

### CODE REVIEW

File: content/docs/overview/faq.md [hunk 1]
```diff
@@ -26,6 +26,25 @@ mirrord also has specific support for the following languages that don't use lib
...
+If an agent pod's status is `Running`, it means mirrord is probably still running locally as well. Once you
+terminate the local process, the agent pod's status should change to `Completed`.
+
+On clusters with Kubernetes version v1.23 or newer, agent pods are supposed to be
+[cleaned up automatically](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/) after a
+[configurable](/docs/overview/configuration/#agent-ttl) TTL from completion.
+If on your cluster mirrord agent pods are lingering for
+longer than the configured TTL after completion,
+[please open an issue on GitHub](
+https://github.com/metalbear-co/mirrord/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=Agent%20pods%20lingering%20after%20completion
```
Comment (t4lz) [inline content/docs/overview/faq.md:41]: Is the issue title ok? "Agent pods lingering after completion"

File: content/docs/overview/faq.md [hunk 2]
```diff
@@ -26,6 +26,25 @@ mirrord also has specific support for the following languages that don't use lib
...
 If you have any restrictions for pulling external images inside your cluster, you have to allow pulling of ghcr.io/metalbear-co/mirrord image.
+
+## Old mirrord agent pods are not getting deleted after the mirrord run is completed, what can I do?
+
+If an agent pod's status is `Running`, it means mirrord is probably still running locally as well. Once you
+terminate the local process, the agent pod's status should change to `Completed`.
+
+On clusters with Kubernetes version v1.23 or newer, agent pods are supposed to be
+[cleaned up automatically](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/) after a
+[configurable](/docs/overview/configuration/#agent-ttl) TTL from completion.
```
Comment (aviramha) [inline content/docs/overview/faq.md:37]: I would add that by default that immediately.

File: content/docs/overview/faq.md (hunk 1 above)
Comment (aviramha) [reply content/docs/overview/faq.md:41]: Nice

**Review by aviramha — APPROVED** [review]

File: content/docs/overview/faq.md (hunk 2 above)
Comment (t4lz) [reply content/docs/overview/faq.md]: Like this? https://github.com/metalbear-co/mirrord.dev/pull/139/commits/5137ed8d6b67a5a6354429b76160c60c79d83c7f

File: content/docs/overview/faq.md [hunk 3]
```diff
@@ -26,6 +26,25 @@ mirrord also has specific support for the following languages that don't use lib
 No, mirrord doesn't install anything on the cluster, nor does it have any persistent state. It does spawn a short-living pod/container to run the proxy, which is automatically removed when mirrord exits. mirrord works using the Kubernetes API, and so the only prerequisite to start using mirrord is to have kubectl configured for your cluster.
 
 If you have any restrictions for pulling external images inside your cluster, you have to allow pulling of ghcr.io/metalbear-co/mirrord image.
+
+## Old mirrord agent pods are not getting deleted after the mirrord run is completed, what can I do?
+
+If an agent pod's status is `Running`, it means mirrord is probably still running locally as well. Once you
+terminate the local process, the agent pod's status should change to `Completed`.
+
+On clusters with Kubernetes version v1.23 or newer, agent pods are supposed to be
```
Comment (eyalb181) [inline content/docs/overview/faq.md:35]:
```suggestion
On clusters with Kubernetes version v1.23 or newer, agent pods are
```

File: content/docs/overview/faq.md [hunk 4]
```diff
@@ -26,6 +26,25 @@ mirrord also has specific support for the following languages that don't use lib
...
+[cleaned up automatically](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/) after a
```
Comment (eyalb181) [inline content/docs/overview/faq.md:36]: Rephrase to: automatically cleaned up immediately (or after a [configurable TTL](link)).

File: content/docs/overview/faq.md [hunk 5]
```diff
@@ -26,6 +26,25 @@ mirrord also has specific support for the following languages that don't use lib
...
+[configurable](/docs/overview/configuration/#agent-ttl) TTL from completion (one second by default).
+If on your cluster mirrord agent pods are lingering for
```
Comment (eyalb181) [inline content/docs/overview/faq.md:38]:
```suggestion
If your cluster is v1.23 or higher and mirrord agent pods are not being cleaned up automatically
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/docs/overview/faq.md (hunk 4 above)
Comment (t4lz) [reply content/docs/overview/faq.md]: Done in https://github.com/metalbear-co/mirrord.dev/pull/139/commits/b29b638ecd69c479313efc6a706f760296f4191e

**Review by eyalb181 — CHANGES_REQUESTED** [review]

File: content/docs/overview/faq.md [hunk 6]
```diff
@@ -26,6 +26,25 @@ mirrord also has specific support for the following languages that don't use lib
...
+## Old mirrord agent pods are not getting deleted after the mirrord run is completed, what can I do?
+
+If an agent pod's status is `Running`, it means mirrord is probably still running locally as well. Once you
+terminate the local process, the agent pod's status should change to `Completed`.
+
+On clusters with Kubernetes version v1.23 or higher, agent pods are
+[automatically cleaned up](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/)
+immediately (or after a [configurable TTL](/docs/overview/configuration/#agent-ttl)).
+If your cluster is v1.23 or higher and mirrord agent pods are not being cleaned up automatically
+longer than the configured TTL after completion,
```
Comment (t4lz) [inline content/docs/overview/faq.md:39]:
```suggestion
If your cluster is v1.23 or higher and mirrord agent pods are not being cleaned up automatically,
```

[the remark this answers is not in the document]
Comment (t4lz) [reply content/docs/overview/faq.md]: removed (https://github.com/metalbear-co/mirrord.dev/pull/139/commits/5f6090d699b8aa666f2b9ad13f94027edb877bef)

File: content/docs/overview/faq.md [hunk 7]
```diff
@@ -26,6 +26,24 @@ mirrord also has specific support for the following languages that don't use lib
 No, mirrord doesn't install anything on the cluster, nor does it have any persistent state. It does spawn a short-living pod/container to run the proxy, which is automatically removed when mirrord exits. mirrord works using the Kubernetes API, and so the only prerequisite to start using mirrord is to have kubectl configured for your cluster.
 
 If you have any restrictions for pulling external images inside your cluster, you have to allow pulling of ghcr.io/metalbear-co/mirrord image.
+
+## Old mirrord agent pods are not getting deleted after the mirrord run is completed, what can I do?
```
Comment (eyalb181) [inline content/docs/overview/faq.md:30]: Please move it to the bottom of the list

Comment (t4lz) [reply content/docs/overview/faq.md]: Done.

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**